pyacq/core/stream/ringbuffer.py

```diff
@@ -215,8 +215,15 @@ def get_data(self, start, stop, copy=False, join=True):
         copied = False
         
         if self.double:
-            start_ind = start % bsize
-            stop_ind = start_ind + (stop - start)
+            # This do not work when get_data(-10, 50) meaning stop=50 length=60 (start=stop-length)
```

Comment:
I think I am not clear again.
If you want to compute FFT of N last sample. The natural is to do ```get_data(head-N, head)```
But if the head index is before N because we are at the very begining so ```head<N```. We don't want to have extra line of code to handle this. So with this tric, when start= head-N and start <0. the get_data give anyway N point because the modulo is done on stop which is alway >0 and not on the start.